### PR TITLE
Unpublished

### PR DESCRIPTION
**Pull request by tylerlee**
If they're an admin let them see pending events

### CODE REVIEW

File: scripts/components/events/calendar.es6 [hunk 1]
```diff
@@ -25,13 +25,16 @@ export default React.createClass({
   fetch: function () {
     if (this.state.isLoading || this.state.error) return;
     this.update({isLoading: {$set: true}, error: {$set: null}});
-    fetch({
+    var options = {
```
Comment (caseywebdev) [inline scripts/components/events/calendar.es6]: Prefer `const` wherever we can.

File: scripts/components/events/calendar.es6 [hunk 2]
```diff
@@ -25,13 +25,16 @@ export default React.createClass({
...
       after: this.getStartMom().toISOString(),
       before: this.getEndMom().toISOString(),
       ranges: this.state.ranges,
       events: this.state.allEvents,
-      url: this.props.eventsUrl
-    }, this.handleFetch);
+      url: this.props.eventsUrl,
```
Comment (caseywebdev) [inline scripts/components/events/calendar.es6]: Kill the trailing comma.

**Comment by caseywebdev** [comment]
Some style stuff, then LGTM

**Comment by tylerlee** [comment]
merge after https://github.com/orgsync/orgsync/pull/9062

**Pull request merged**